src/components/DateRangePicker/index.js

```diff
@@ -16,9 +18,15 @@ class DateRangePicker extends Component {
   }
   render() {
     const { focusedRange } = this.state;
+    // All calendar calculations are based on the current date aka `now`. However, operations with dayjs() objects
```

Comment:
At first, I tried exposing a `timezone` property to do `dayjs().tz()` calls but the component was extremely laggy; as the main focus of having a timezone is the current date, and the calendar calculations are mostly based on this (except `ranges` provided via the `ranges` property, _which if contains timezones will make the component laggy too_), I found that providing a `now` property is a better approach.